core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/LogicalOp.scala

```diff
@@ -241,4 +244,9 @@ abstract class LogicalOp extends PortDescriptor with Serializable {
     )
   }
 
+  @JsonProperty
+  @JsonSchemaTitle("")
+  @JsonPropertyDescription("Add dummy property if needed")
+  var dumProp: List[DummyProperties] = List()
```

Comment:
It does not look good to have dummy properties showing on normal operators that do not need this. Please consider if it is possible to hide them in the frontend if an operator does not have dummy properties.